src/style.css

```diff
@@ -1 +1,57 @@
-/* add styles here */
+/* Reset browser default margin */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Wrapper for stars */
+.stars-container {
+  display: flex;
+  justify-content: center; /* Centers the container horizontally */
+  align-items: center; /* Centers the container vertically if height is defined */
+  padding: 20px; /* Adds spacing between the container and the edges of the page */
+  background-color: #f0f0f0; /* Optional background for better visibility */
+}
+
+/* General star block styles */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../layout_stars/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+/* Star activation */
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The selector '.stars--0 .stars__star:nth-child(-n + 0)' will not apply any active star styles because '-n + 0' does not select any elements. Consider changing this to '.stars--0 .stars__star:nth-child(1)' if you want to activate the first star, or adjust according to your design requirements.